bnd.bnd

```diff
@@ -1,8 +1,7 @@
 Bundle-Name: ${project.name}
 Bundle-Version: ${project.version}
 Bundle-SymbolicName: ${project.groupId}.${project.artifactId}
-Bundle-DocURL: ${project.url}
-
-Bundle-Activator: life.qbic.example.Activator
-
-Export-Package: life.qbic.example
+Bundle-URL: ${project.url}
+Export-Package: life.qbic.*;version="${project.version}"
+Import-Package: !org.everit.json.schema.*,!org.json.*,!org.xml.sax,\\
```

Comment:
should this be two backslashes? I would expect one

Reply:
That's the syntax that worked to successfully create the bundle :)